tests/integration/api_v3/rpc/transactionsDryRun.test.js

```diff
@@ -136,7 +169,7 @@ describe('Method post.transactions.dryrun', () => {
 		expect(result.meta).toMap(metaSchema);
 	});
 
-	xit('Returns proper response (Invalid) for duplicate transaction', async () => {
+	xit('should return proper response (Invalid) for duplicate transaction', async () => {
```

Comment:
Why do we have this testcase disabled?
```suggestion
	xit('should return proper response (invalid) for duplicate transaction', async () => {
```